native/monogame/sdl/MGP_sdl.cpp

```diff
@@ -257,6 +257,8 @@ MGMonoGamePlatform MGP_Platform_GetPlatform()
 {
 #if MG_VULKAN
     return MGMonoGamePlatform::DesktopVK;
+#elif MG_DIRECTX12
+    return MGMonoGamePlatform::Windows;
```

Comment:
So in this case we just use SDL on desktop, so we use Windows as the platform.

But for Vulkan i had added a platform `DesktopVK`.

I think we need to discuss and rethink what is a platform.